modules/apps/journal/journal-web/src/test/java/com/liferay/journal/web/internal/util/JournalUtilTest.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.journal.web.internal.util;
+
+import com.liferay.diff.DiffVersion;
+import com.liferay.diff.DiffVersionsInfo;
+import com.liferay.journal.model.JournalArticle;
+import com.liferay.journal.service.JournalArticleServiceUtil;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.OrderByComparator;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+/**
+ * @author Gergely Szalay
+ */
+public class JournalUtilTest {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
+		LiferayUnitTestRule.INSTANCE;
+
+	@Before
+	public void setUp() {
+		_setUpJournalArticleServiceUtil();
+	}
+
+	@After
+	public void tearDown() {
+		_journalArticleServiceUtilMockedStatic.close();
+	}
+
+	@Test
+	public void testGetDiffVersionsInfo() {
+		DiffVersionsInfo diffVersionsInfo = JournalUtil.getDiffVersionsInfo(
+			RandomTestUtil.randomLong(), RandomTestUtil.randomString(), 1.0,
+			1.3);
+
+		List<DiffVersion> diffVersions = diffVersionsInfo.getDiffVersions();
+
+		for (DiffVersion diffVersion : diffVersions) {
```

Comment:
Personal suggestion: 
Maybe this will be simpler and more readable if we use different asserts instead of a for and if .

But can be forwarded :)